.github/workflows/build.yml

```diff
@@ -353,6 +353,10 @@ jobs:
           path: dist
       - name: Generate checksum file
         run: cd dist && sha256sum * > "k6-${VERSION}-checksums.txt"
+      - name: Anchore SBOM Action
+        uses: anchore/sbom-action@v0.12.0
+        with:
+          artifact-name: ${{ github.event.repository.name }}-spdx.json
```

Comment:
I wonder if the asset detection wilkl "work" as such I do think it might be better if we actually output it to the `dist` directory and let teh `create release` below add it to the assets :thinking: 

Maybe also setting a bunch of the other options to false as action should not try to upload anything on it 's own now that we have it as part of the release

Reply:
Hi @mstoykov, I've just updated the **Anchore SBOM Action** to add the SBOM output to `dist` directory in my latest commit. It then be processed by the `create release` step to add to the release assets.